docs/responsive-ui-dev.md

```diff
@@ -0,0 +1,11 @@
+# Developing our Responsive UI
```

Comment:
Are these instructions still applicable now that the folders are merged?  Suggesting that we remove, or add some context about responsive development instead.